src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -45,9 +45,9 @@
             <a
               id="sourceRepository"
               data-cy="sourceRepository"
-              [href]="workflow?.providerUrl | versionProviderUrl: (isPublic ? selectedVersion?.name : '')"
+              [href]="workflow?.providerUrl | versionProviderUrl: (isPublic ? selectedVersion?.name : '') + '/' + sourceCodeFile"
             >
-              {{ workflow?.providerUrl | urlDeconstruct: (isPublic ? selectedVersion?.name : '') }}
+              {{ sourceCodeFile }}
```

Comment:
I wonder if we should keep the text as before because looking at it locally, the file name looks out of place next to `Source Code`.

![image](https://user-images.githubusercontent.com/25287123/215888192-33a0a312-aaf4-4ca4-952c-6df1c021c45f.png)

> Also, some workflows had very long paths so the text of the link was changed to only display the file name.

Did it not wrap properly with the long paths?

Reply:
> Did it not wrap properly with the long paths?

think it wrapped but looked silly (i.e. wrapped more than once with some examples)

Reply:
It wraps but it looked a bit weird when the link got too long. Here was an example that I encountered
![image](https://user-images.githubusercontent.com/61166764/215893634-881537ff-9376-4996-a2d4-39e126fbb3c0.png)

Reply:
Perhaps a compromise is to truncate the text if it's more than one line? Looks something like this:

![image](https://user-images.githubusercontent.com/25287123/216075773-cb37dc99-6f0d-4125-a3f2-3b69e9e3c716.png)

I used the the `truncate-text-1` class from [here](https://github.com/dockstore/dockstore-ui2/blob/7dd11b7dd5d46c56ad7e6016030b4fc023a1d338/src/styles.scss#L1201)

Reply:
(personally ok either way as long as the mouse-over URL is good)